composites/Plugin/ContentAnalysis/components/ContentAnalysis.js

```diff
@@ -145,6 +145,45 @@ class ContentAnalysis extends React.Component {
 		} );
 	}
 
+	/**
+	 * Render the language notice. Provides a link to a setting page in case of administrator, a notice to contact an
+	 * administrator otherwise.
+	 *
+	 * @returns {ReactElement} The rendered language notice.
+	 */
+	renderLanguageNotice() {
+		let showLanguageNotice = this.props.showLanguageNotice;
+		let canChangeLanguage = this.props.canChangeLanguage;
+		if( ! showLanguageNotice ) {
+			return null;
+		}
+		if( canChangeLanguage ) {
+			return (
+				<LanguageNotice>
+					<FormattedMessage
+						id="content-analysis.language-notice"
+						defaultMessage="Your site language is set to {language}."
+						values={ { language: <strong>{ this.props.language }</strong> } } />
+					<ChangeLanguageLink href={ this.props.changeLanguageLink }>
+						{ this.props.intl.formatMessage( messages.languageNoticeLink ) }
+					</ChangeLanguageLink>
+				</LanguageNotice>
+			);
+		}
+		return (
+			<LanguageNotice>
+				<FormattedMessage
+					id="content-analysis.language-notice"
+					defaultMessage="Your site language is set to {language}."
+					values={ { language: <strong>{ this.props.language }</strong> } } />
+				{ " " }
```

Comment:
minor: instead of explicitly using a space, I'd consider to use a translatable string that merges the 2 existing ones